### PR TITLE
Legacy Python certificate validation fixed

### PR DESCRIPTION
**Pull request by PetzJohannes**
##### SUMMARY
Fix Python 2.7.5 problem with ssl certificate validation.
See: https://github.com/ansible-collections/community.general/pull/79

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
plugins/modules/packaging/os/rhn_channel.py

### CODE REVIEW

**Comment by Andersson007** [comment]
please add a changelog fragment https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to

**Comment by PetzJohannes** [comment]
@Andersson007 changelog fragment added.

**Comment by Andersson007** [comment]
@vincentvdk hey, as the module's author, could you please take a look?

File: changelogs/fragments/470-spacewalk-legacy-python-certificate-validation.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - rhn_channel.py (spacewalk) - Python 2.7.5 fails if the certificate should not be validated. Fixed this by creating the correct ssl_context. (https://github.com/ansible-collections/community.general/pull/470)
```
Comment (Andersson007) [inline changelogs/fragments/470-spacewalk-legacy-python-certificate-validation.yaml:2]:
```suggestion
  - rhn_channel - Python 2.7.5 fails if the certificate should not be validated. Fixed this by creating the correct ssl_context (https://github.com/ansible-collections/community.general/pull/470).
```

**Comment by Andersson007** [comment]
@vincentvdk could you look at the changes please

**Comment by Andersson007** [comment]
@PetzJohannes sorry for the delayed reply. I've just emailed the author

**Comment by Andersson007** [comment]
i closed and reopened the PR to trigger the tests

File: changelogs/fragments/470-spacewalk-legacy-python-certificate-validation.yaml (hunk 1 above)
Comment (Andersson007) [reply changelogs/fragments/470-spacewalk-legacy-python-certificate-validation.yaml:2]: any updates on this? https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs-how-to

**Review by vincentvdk — APPROVED** [review]

File: changelogs/fragments/470-spacewalk-legacy-python-certificate-validation.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - rhn_channel - Python 2.7.5 fails if the certificate should not be validated. Fixed this by creating the correct ssl_context (https://github.com/ansible-collections/community.general/pull/470).
```
Comment (felixfontein) [inline changelogs/fragments/470-spacewalk-legacy-python-certificate-validation.yaml:2]:
```suggestion
  - rhn_channel - Python 2.7.5 fails if the certificate should not be validated. Fixed this by creating the correct ``ssl_context`` (https://github.com/ansible-collections/community.general/pull/470).
```

**Comment by felixfontein** [comment]
@PetzJohannes thanks for fixing this!
@Andersson007 @vincentvdk thanks for reviewing this!

**Pull request merged**